### PR TITLE
Bc form design #17

### PR DESCRIPTION
**Pull request by jpham93**
|#412 https://projects.vta.org/projects/copy-center/work_packages/412

This Pull Request updates the business card forms UI. It follows the images that are posted below. The new form has tab navigation (form validation is still applied i.e. can't navigate if required fields are not filled out).

**Requirements:**
- [x] The form should be segmented into 3 steps: Information, Contact, and Request
- [x] The form sections will be separated into **tabs**.
- [x] Have basic form validation for **required** fields.
- [x] Create visual confirmation for fields that have been filled out.
- [x] Users can navigate with next and previous buttons **OR** tabs.

**Note:**
The location of the button was moved towards the bottom. The original location did not feel intuitive as a user. Also, please test on mobile screen widths for this review.

**Mockups:**
bc_contact
bc_submit
bc_YourInfo

### CODE REVIEW

File: wp-content/themes/twentytwenty-child/style.css [hunk 1]
```diff
@@ -11,19 +11,52 @@ Version: 1.0
...
-          rgba(25,95,145,1) 35%,
-          rgba(80,119,161,1) 55%,
-          rgba(255,255,255,1) 100%
+          rgba(25,95,145,1) 47%,
+          rgba(80,119,161,1) 79%,
+          rgba(138,164,192,1) 100%
   );
 
+  /* Borders */
+  --thin-light-gray:  1px solid var(--light-gray);
```
Comment (bijanmmarkes) [inline wp-content/themes/twentytwenty-child/style.css:35]: Any chance to integrate SASS? :)

--- Nice use of variables for global colors! :+1:

File: wp-content/themes/twentytwenty-child/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * Business Card Tabs Function
+ *
+ * creates event listeners for each tab click
+ * @param $ - jQuery Selector
+ */
+const bc_tabs = ($) => {
+
+  // BC Tab 1: Your Information
+  $('#gf_step_4_1').live('click', () => bc_select_tab($, '1'));
```
Comment (bijanmmarkes) [inline wp-content/themes/twentytwenty-child/js/scripts.js:24]: Does this prevent clicking tabs before the tab they're currently on is complete?

**Review by bijanmmarkes — APPROVED** [review]
Overall looks good, hopefully, at some point, we can integrate SASS. :+1:

File: wp-content/themes/twentytwenty-child/style.css (hunk 1 above)
Comment (jpham93) [reply wp-content/themes/twentytwenty-child/style.css:35]: I think SASS will be an excellent idea! A lot of the selectors could and _should_ be nested. The structure will be a lot easier to follow too.

File: wp-content/themes/twentytwenty-child/js/scripts.js (hunk 2 above)
Comment (jpham93) [reply wp-content/themes/twentytwenty-child/js/scripts.js:24]: Gravity Forms takes care of that on the backend, so form validation works the same on tabs as it does with **Back** and **Next** buttons.

**Pull request merged**